support-workers/src/test/scala/com/gu/support/workers/JsonFixtures.scala

```diff
@@ -534,7 +534,8 @@ object JsonFixtures {
               "Id": "0036E00000VlOPDQA3",
               "AccountId": "0016E00000f17pYQAQ"
           },
-          "acquisitionData": $acquisitionData
+          "acquisitionData": $acquisitionData,
+          $salesforceContactsJson
```

Comment:
this was because the deserialiser was expecting this field too since gifting was introduced, and this fixture was only used in IT tests. https://github.com/guardian/support-frontend/pull/1909/files#diff-2c58c36244db5235e18da3b61dd55736R20